core/integration/main_test.go

```diff
@@ -462,6 +462,14 @@ func InitializeScenario(s *godog.ScenarioContext) {
 		execsetup.assetsEngine.SetPermissive()
 		return nil
 	})
+
```

Comment:
Is this the bit I added a while back? Because if it is, through some other PR a while back I added the position status bit to the existing positions table, so this probably isn't needed anymore.

Reply:
I am not sure which other PRs you meant, could you remove this part, and then pull develop, and then rerun the feature tests?